dist/selectivity-full.js

```diff
@@ -360,7 +360,8 @@ $.extend(SelectivityDropdown.prototype, {
         } else {
             $backdrop = $('<div>').addClass('selectivity-backdrop');
 
-            $('body').append($backdrop);
+            this.$el.prepend($backdrop);
+            this.$el.children().not('.selectivity-backdrop').css('z-index',9999);
```

Comment:
May this can be fixed in the css. But I don't know which css class need be updated. So just use the javascript.